### PR TITLE
Jennyf/ios broker

### PR DESCRIPTION
**Pull request by jennyf19**
Registered a new app and broker is working w/this application. Once merged, i can reference it i the wiki if we want.

### CODE REVIEW

File: UserDetailsClient/UserDetailsClient.iOS/packages.config [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <package id="AsyncUsageAnalyzers" version="1.0.0-alpha003" targetFramework="xamarinios10" developmentDependency="true" />
   <package id="Microsoft.Identity.Client" version="4.5.1" allowedVersions="[4,5)" targetFramework="xamarinios10" />
```
[the remark this answers is not in the document]
Comment (jennyf19) [reply UserDetailsClient/UserDetailsClient.iOS/packages.config]: @jmprieur that's crazy...I'll explain in person but the line below didn't show up when I pushed the changes. There should be one reference

[the remark this answers is not in the document]
Comment (jmprieur) [reply UserDetailsClient/UserDetailsClient.iOS/packages.config]: so many things are crazy, @jennyf19 :) !

**Comment by jennyf19** [comment]
@jmprieur we could do two folders, it just that the code changes are pretty minimal, but you decide, you are closer to customers. Whatever is easier. Or maybe that's something we do when we have Android broker support as well? For now, this only works on iOS.
Yes, both redirect URIs have been registered.

**Comment by jmprieur** [comment]
My worry is that we know that a lot of customers just copy/paste the code. so we would want to have the simplest possible code. Alternatively we could use #ifdefs ? but I'm not too sure. @henrik-me what do you think?

Or do we want, like iOS native, to promote broker only?

I forgot. We also need to update the README.md

**Comment by jennyf19** [comment]
@jmprieur okay, makes sense. Let's do two folders. I'll make the changes tomorrow and update the readme as well. Good point. And I'll change the stuff to not use the package reference anymore too

**Comment by jmprieur** [comment]
@jennyf19 : we might want to re-create the solution from scratch. I believe @bgavrilMS had proposed a PR in this direction (https://github.com/Azure-Samples/active-directory-xamarin-native-v2/pull/69), which we have never merged (my bad :( )

**Comment by jennyf19** [comment]
@jmprieur yeah, i thought of doing it from scratch when you mentioned having two solutions. I will take a look at what @bgavrilMS  has already started. thanks.

**Pull request closed without merging**